cliquet/tests/test_authorization.py

```diff
@@ -73,6 +73,24 @@ def test_attributes_are_none_with_non_resource_requests(self):
         self.assertIsNone(context.resource_name)
         self.assertIsNone(context.check_permission)
 
+    def test_service_collection_path_is_correct(self):
```

Comment:
It is not the test, what you describe is: https://github.com/mozilla-services/cliquet/pull/303/files#diff-d6ddebfba0cf4aea5bdbcb72d77b3610R47

Here it is more, ``test_service_collection_path_resolve_to_current_collection_path``